src/DataPolicyAccessControlHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\gdpr_consent;
+
+use Drupal\Core\Entity\EntityAccessControlHandler;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Access\AccessResult;
+
+/**
+ * Access controller for the Data policy entity.
+ *
+ * @see \Drupal\gdpr_consent\Entity\DataPolicy.
+ */
+class DataPolicyAccessControlHandler extends EntityAccessControlHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function access(EntityInterface $entity, $operation, AccountInterface $account = NULL, $return_as_object = FALSE) {
+    $access = parent::access($entity, $operation, $account, $return_as_object);
+    $x = 1;
```

Comment:
@OH17L this seems out of place. Perhaps remove it?

Reply:
@bramtenhove, yes, I will delete it.